casual/casual-network/src/main/java/se/laz/casual/network/outbound/NettyNetworkConnection.java

```diff
@@ -92,11 +96,16 @@ public static NetworkConnection of(final NettyConnectionInformation ci, final Ne
         ch.closeFuture().addListener(f -> handleClose(networkConnection, errorInformer));
         DomainId id = networkConnection.throwIfProtocolVersionNotSupportedByEIS(ci.getDomainId(), ci.getDomainName());
         networkConnection.setDomainId(id);
-        if(networkConnection.getProtocolVersion() == ProtocolVersion.VERSION_1_1 || networkConnection.getProtocolVersion() == ProtocolVersion.VERSION_1_2)
+        if(networkConnection.isProtocolVersionOneOneOrOneTwo())
```

Comment:
Could `networkConnection.isProtocolVersionOneOneOrOneTwo()` and `networkConnection.isProtocolVersionOneTwo()` be broken up into specific feature checks, like `networkConnection.protocolSupportsDomainDisconnect()` and `networkConnection.protocolSupportsDomainTopologyChange()`. If casual adds more protocol levels methods don't need renaming and we won't get methods like isProtocolVersionOneOneOrOneTwoOrOneThree and so on

Reply:
Done